packages/components/pages/f-checkout/package.json

```diff
@@ -3,7 +3,7 @@
   "description": "Fozzie Checkout - Fozzie Checkout Component",
   "version": "4.4.0",
   "main": "dist/f-checkout.umd.min.js",
-  "maxBundleSize": "130kB",
+  "maxBundleSize": "140kB",
```

Comment:
Bundlesize increased as the dependencies use the new version of f-services which is slightly bigger.